app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+
+    source, destination = command.split()[1:]
+
+    source_parts = source.rsplit("/", 1)
+    destination_parts = destination.rsplit("/", 1)
+
+    if len(destination_parts) > 1:
```

Comment:
You are using rsplit with max splits 1 so this condition will never match.
Chech your function locally.